web/users_test.go

```diff
@@ -281,21 +276,6 @@ func TestUpdateUserFailures(t *testing.T) {
 	// context with user u (non-admin user); can only change its own name etc
 	ctx := qtest.WithUserContext(context.Background(), u)
 
-	// trying to demote current adminUser by setting IsAdmin to false
-	nameChangeRequest := &qf.User{
-		ID:        wantAdminUser.ID,
-		IsAdmin:   false,
-		Name:      "Scrooge McDuck",
-		StudentID: "99",
-		Email:     "test@test.com",
-		AvatarURL: "www.hello.com",
-	}
-	// current user u (non-admin) is in the ctx and tries to change adminUser
-	_, err := ags.UpdateUser(ctx, nameChangeRequest)
```

Comment:
Just checking my understanding. This is no longer the right place to check this, since an RPC call would never reach this point because the interceptor would have stopped it.

Reply:
Correct, it would never reach this point with interceptors. Moreover, in any test in the `web` package any user can call any method because there are no access control checks inside the methods.